python-lib/config.py

```diff
@@ -82,6 +82,29 @@ def get_hdfs_location(dataset_config: Mapping[AnyStr, Any], sf_stage_name: AnySt
     return f'{sf_stage_name}{path}/'
 
 
+def get_table_schema(sf_table_name: AnyStr, executor: Any) -> List[Mapping[AnyStr, AnyStr]]:
```

Comment:
I would suggest making this `get_table_schema_sql` and just return the query like all other methods in this file. It'll make it easier to unit test and keep it consistent with the structure of the rest of the plugin.